src/ddceler/DDcelerTMI.cc

```diff
@@ -0,0 +1,71 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file ddceler/DDcelerTMI.cc
+//---------------------------------------------------------------------------//
+#include "DDcelerTMI.hh"
+
+#include <DDG4/Factories.h>
+#include <QGSP_BERT.hh>
+
+#include "celeritas/field/FieldDriverOptions.hh"
+#include "celeritas/inp/Field.hh"
+
+using TMI = celeritas::TrackingManagerIntegration;
+
+namespace dd4hep
+{
+namespace sim
+{
+//---------------------------------------------------------------------------//
+celeritas::SetupOptions DDcelerTMI::makeOptions()
+{
+    celeritas::SetupOptions opts;
+
+    // NOTE: these numbers are appropriate for CPU execution and can be set
+    // through the UI using `/celer/`
+    opts.max_num_tracks = 2024;
+    opts.initializer_capacity = 2024 * 128;
+    // Celeritas does not support EmStandard MSC physics above 200 MeV
+    opts.ignore_processes = {"CoulombScat"};
```

Comment:
Would be useful to include in the steeringFile.py the setting of these options through the pythonic interface. Nobody I know uses the geant4 command macros with dd4hep (if they can avoid it).